newspack-theme/sass/site/primary/_archives.scss

```
@@ -31,6 +31,10 @@
 
 	.post-thumbnail {
 		margin: 0 0 #{0.5 * structure.$size__spacing-unit};
+
+		figcaption {
+			margin-top: 8px;
```

Another nitpick -- I think we might be able to get away with a smaller font size here, at least for desktop sized screens -- it feels a bit big with the photos on the left, though the default size is much better on mobile.

```suggestion
			margin-top: 8px;
			@include utilities.media( tablet ) {
				font-size: var(--newspack-theme-font-size-xxs);
			}
```